src/domain/gift-cards/manage/sections/information.tsx

```diff
@@ -104,30 +109,22 @@ const Information: React.FC<InformationProps> = ({ giftCard }) => {
       ]}
     >
       <div className="flex flex-col space-y-6">
-        <div className="flex space-x-8">
-          <div className="flex flex-col w-1/2 space-y-4">
-            <Input
-              label="Name"
-              name="title"
-              placeholder="Add name"
-              defaultValue={giftCard?.title}
-              ref={register}
-            />
-            <Input
-              label="Subtitle"
-              name="subtitle"
-              placeholder="Add a subtitle"
-              defaultValue={giftCard?.subtitle}
-              ref={register}
-            />
-          </div>
+        <div className="grid grid-cols-2 gap-large">
           <Input
+            label="Name"
```

Comment:
question: should the field be ;marked as required as well? (general question) as already mentioned else where 😂